nuimo/src/main/kotlin/com/senic/nuimo/NuimoBluetoothController.kt

```diff
@@ -44,7 +44,9 @@ class NuimoBluetoothController(bluetoothDevice: BluetoothDevice, private val con
             // If there is still a reference to gatt we should close it
             gatt?.close()
             reset()
-            gatt = createConnection()
+            try {
+                gatt = createConnection()
+            } catch (e: IllegalStateException) { notifyListeners { it.onFailToConnect(REASON_GATT_ERROR) }}
```

Comment:
According to https://android.googlesource.com/platform/frameworks/base/+/refs/heads/master/core/java/android/bluetooth/BluetoothDevice.java#1594
we know here that BLE is not supported. Let's encode it in the error code.

E.g. `ERROR_BLE_NOT_SUPPORTED`